.SRCINFO

```diff
@@ -1,8 +1,6 @@
-# Generated by makepkg 4.2.1
```

Comment:
The .SRCINFO file is generated automatically by makepkg.  The new deployment method makes this obsolete, so if you could remove this file and add it to the .gitignore, that would be great.

Reply:
Resolved!